api/graphql/types/application_section/filtersets.py

```diff
@@ -116,3 +125,11 @@ def order_by_preferred_unit_name_en(qs: ApplicationSectionQuerySet, desc: bool)
     @staticmethod
     def order_by_preferred_unit_name_sv(qs: ApplicationSectionQuerySet, desc: bool) -> QuerySet:
         return qs.order_by_preferred_unit_name(lang="sv", desc=desc)
+
+    @staticmethod
+    def order_by_has_allocations(qs: ApplicationSectionQuerySet, desc: bool) -> QuerySet:
+        return qs.order_by(models.OrderBy(L(allocations__gt=0), descending=desc))
+
+    @staticmethod
+    def order_by_allocations(qs: ApplicationSectionQuerySet, desc: bool) -> QuerySet:
+        return qs.order_by(models.OrderBy(L("allocations"), descending=desc))
```

Comment:
I was about to comment that these are essentially the same, but I guess `order_by_has_allocations` might be useful if user wants to use multiple `order_by`s at the same time 😄 

Might be good to add a comment for the reason, as these looked like duplicates and the reason was not clear instantly

Reply:
I think their meaning doesn't require a comment - it's quite clear from just reading the code. But for what purpose they are for, I just implemented what was on the ticket:

> ..Lisätään rajapintaan boolean-tyyppinen suodatin sille, onko hakemuksen osalla tämä tieto > 0 vai ei ,ja lisäksi mahdollisuus sortata sekä sen, että allokaatioiden lukumäärän avulla...

@vergama Maybe has some insight as to why both are needed?